documentation/modules/exploit/windows/http/trend_micro_officescan_widget_exec.md

```diff
@@ -0,0 +1,61 @@
+## Vulnerable Application
+
+This module exploits the authentication bypass and command injection vulnerability together. Unauthenticated users can execute a terminal command under the context of the web server user.
+
+Trend Micro Officescan product have widget feature which is implemented with PHP. Talker.php takes ack and hash parameter but don't validate these values, which leads to an authentication bypass for widget. Proxy.php files under the mod TMCSS folder takes multiple parameter but the process does not properly validate a user-supplied string before using it to execute a system call. Due to combination of these vulnerabilities, unauthenticated users can execute a terminal command under the context of the web server user.
+
+**Vulnerable Application Installation Steps**
+
+1. Open following URL [http://downloadcenter.trendmicro.com/](http://downloadcenter.trendmicro.com/)
+2. Find "OfficeScan" and click.
+3. At the time of writing this documentation, you must see "osce-xg-win-en-gm-b1315.exe" next to Download button.
+4. Click to the download button and complete installation of ISO.
+5. Install the downloaded file on Windows operating system. (Tested with Windows 7)
+
+If you don't see an affected version of OfficeScan, you can try to download it directly from following URL.
+
+[ftp://download.trendmicro.com/products/officescan/XG/osce_xg_win_en_gm_b1315.exe](ftp://download.trendmicro.com/products/officescan/XG/osce_xg_win_en_gm_b1315.exe)
+
+## Verification Steps
+
+A successful check of the exploit will look like this:
+
+- [ ] Start `msfconsole`
+- [ ] `use exploit/windows/http/trendmicro_officescan_exec`
+- [ ] Set `RHOST`
+- [ ] Set `LHOST`
+- [ ] Run `check`
+- [ ] **Verify** that you are seeing `The target appears to be vulnerable.`
+- [ ] Run `exploit`
+- [ ] **Verify** that you are seeing `PHPSESSIONID` value.
+- [ ] **Verify** that you are getting `meterpreter` session.
+
+## Scenarios
+
+```
+msf > use exploit/windows/http/trendmicro_officescan_exec
```

Comment:
Going to change it to `trendmicro_officescan_widget_exec`. I'm just waiting a full review so I can fix them with a single commit.

Reply:
Yeah I didn't report it because I wouldn't get paid. It's how it goes sometimes.

Reply:
But you also found it, so congrats!